keras/backend/common.py

```diff
@@ -63,27 +161,87 @@ def set_image_dim_ordering(dim_ordering):
 
 
 def get_uid(prefix=''):
+    '''
+
+    # Argument
+        prefix: string
+
+    # Return
+        integer (scalar)
+
+    # Example
+    ```
+        >>> a = keras.backend.get_uid()
+        >>> type(a)
+        <type 'int'>
+    ```
+
+    '''
     _UID_PREFIXES[prefix] += 1
     return _UID_PREFIXES[prefix]
 
 
 def reset_uids():
+    '''
+    # Return
+        `None`
+    '''
+
     global _UID_PREFIXES
     _UID_PREFIXES = defaultdict(int)
 
 
 def is_keras_tensor(x):
+    '''Returns if `x` is a Keras tensor,
+    which is equivalent to `tf.placeholder()` or
+    `T.matrix()`, `T.tensor3()`.
```

Comment:
I got it from [the current documentation](https://github.com/fchollet/keras/edit/master/docs/templates/backend.md) Line 79. How should it be fixed?